### PR TITLE
Changed table's cost difference column

### PR DESCRIPTION
**Pull request by jennygaoo**

### CODE REVIEW

File: postal-injection/src/main/webapp/summary_table.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ const ALLOW_HTML = true;
 const BLACK = 'black';
 const DOLLAR_SIGN = '$';
 const HEIGHT = '100%';
-const LIGHT_RED = '#eeffe9';
+const LIGHT_RED = '#ffe8e7';
 const LIGHT_YELLOW = '#fbffc1';
-const LIGHT_GREEN = '#ffe8e7';
+const LIGHT_GREEN = '#eeffe9';
```
Comment (jennygaoo) [inline postal-injection/src/main/webapp/summary_table.js:12]: oops these two colors were accidentally switched, so I changed it in this PR

File: postal-injection/src/main/webapp/summary_table.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ function populateTable(summaryTableData, tableContents) {
   tableContents.addColumn('string', 'Warehouse');
   tableContents.addColumn(
       'number',
-      'Cost Difference Between Postal Injection and Regular Shipping');
+      'Money Saved via Postal Injection');
```
Comment (jennygaoo) [inline postal-injection/src/main/webapp/summary_table.js:93]: "Money" sounds kind of tacky. Do you think "Savings via Postal Injection" would be better (or is this too vague?)

File: postal-injection/src/main/webapp/summary_table.js (hunk 1 above)
Comment (jgao23) [reply postal-injection/src/main/webapp/summary_table.js:12]: probably just do it in a separate PR next time :)

File: postal-injection/src/main/webapp/summary_table.js (hunk 2 above)
Comment (jgao23) [reply postal-injection/src/main/webapp/summary_table.js]: "Savings via Postal Injection" sounds fine to me!

**Comment by jennygaoo** [comment]
> Does this affect Jason's work?

nope

**Pull request merged**